recipes/flex/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "2.6.0" %}
+
+package:
+  name: flex
+  version: {{ version }}
+
+source:
+  fn: flex-{{ version }}.tar.gz
+  url: http://sourceforge.net/projects/flex/files/flex-{{ version }}.tar.gz
+  md5: 5724bcffed4ebe39e9b55a9be80859ec
+
+build:
+  skip: true  # [win]
+  number: 0
+  detect_binary_files_with_prefix: true
```

Comment:
Actually the [opposite]( http://conda.pydata.org/docs/building/meta-yaml.html#detect-binary-files-with-prefix ). Though I wonder if this isn't a hold over from earlier conda build days.